_posts/2016-07-10-chocolate-bread-pudding.markdown

```diff
@@ -0,0 +1,47 @@
+  - plain white sugar (50%)
+  - proper brown sugar, not demerara (25%)
+  - dark chocolate (150%)
+  - a tablespoon of butter
+  - a good splash of vanillin
+  - a pinch of ground clove
+  - quite a bit of ground cinnamon
+allergy_info:
+  - milk, egg, gluten
+directions: |
+  1. Break up the bread into small chunks (think roughly 2" cubes) and cover with milk and cream to soak and soften 
+  2. Once soft, mix in egg, sugar, vanilla, clove, and several tablespoons of cinnamon
+  3. Melt half the chocolate with the tablespoon of butter in the microwave on a half-power setting until it's stirrable.
+  4. Thoroughly mix molten chocolate into the bread mix
+  5. Pour mix in an oven dish so that it sits about an inch below the top of the dish (I find that for 500gr of bread, a 10" x 15" dish works quite well) 
+  6. Coarsely crush up the remaining chocolate and sprinkle on top, then gently press down into the mix
+  7. Finely dust with cinnamon
+  8. Bake at 350F/180C for an hour
+  9. Take out of the oven and have it cool down a little.
+  10. Either eat as a soft, rich, chocolatey pudding immediately, or...
+  11. Wait until cooled down to room temperature, then cover with some kitchen paper towel and aluminium foil and have it compact/cool covered, overnight, then the next day:
+  12. Serve as either an amazing alternative to brownies, or serve as dessert with a crème Anglaise, some ice cream, or anything else you like!
+
+---
+
+It's very tempting to just throw out stale bread, or bread caps, but what if I told you you absolutely shouldn't, and should instead just put them in a bag in the freezer, so that once you accumulate enough of it you can make one of the best desserts of all time?
+
+Chocolate bread pudding is a classic way to "get rid of" old, stale bread, by using it as an ingredient in a delicious pudding recipe. Traditionally served hot out of the oven for a deliciously flavoured, rich, almost molten delight, bread pudding —especially the chocolate variety— can be made even more amazing by having it cool down for a long period of time, giving it time to compact back into an almost pound cake consistency, making it a perfect office snack or easily portioned for dessert. 
+
+It even freezes quite well, keeping for literally months (if packed in an airtight container) and defrosts without a hitch, so trading frozen stale bread for frozen delicious bread cake is well worth it!
+
+Because this is a baking recipe, there are some actual measurements for the ingredients, but they're ratio values: given however much stale bread you're using, use one and a half times as much milk (by weight!), one and a half times as much cream, half as much white sugar, etc. The eggs are a bit of an exception, because portioning eggs by weight is really tricky, so: just round off to whole eggs as you see fit.
+
+![Served with crème Anglaise, chocolate mint ice cream, and fresh mint leaves]({{ site.images_path }}/chocolate-bread-pudding.jpg)
```

Comment:
Perhaps you could replace this one with the squishable photo instead, since this is already used up top (or you could leave it, and add the squishable as a second photo)